pkg/models/cedar_system.go

```diff
@@ -2,13 +2,16 @@ package models
 
 // CedarSystem is the model for a single system that comes back from the CEDAR Core API
 type CedarSystem struct {
-	ID                      string `json:"id"`
-	Name                    string `json:"name"`
-	Description             string `json:"description"`
-	Acronym                 string `json:"acronym"`
-	Status                  string `json:"status"`
-	BusinessOwnerOrg        string `json:"businessOwnerOrg"`
-	BusinessOwnerOrgComp    string `json:"businessOwnerOrgComp"`
-	SystemMaintainerOrg     string `json:"systemMaintainerOrg"`
-	SystemMaintainerOrgComp string `json:"systemMaintainerOrgComp"`
+	ID                      string  `json:"id"`
+	Name                    string  `json:"name"`
+	Description             string  `json:"description"`
+	Acronym                 string  `json:"acronym"`
+	Status                  string  `json:"status"`
+	BusinessOwnerOrg        string  `json:"businessOwnerOrg"`
+	BusinessOwnerOrgComp    string  `json:"businessOwnerOrgComp"`
+	SystemMaintainerOrg     string  `json:"systemMaintainerOrg"`
+	SystemMaintainerOrgComp string  `json:"systemMaintainerOrgComp"`
+	VersionID               string  `json:"versionId"`
+	NextVersionID           *string `json:"nextVersionId"`
+	PreviousVersionID       *string `json:"previousVersionId"`
```

Comment:
I know these are in the response, but I don't think we ever use them (i.e. expose them in the GraphQL) - do we still need them?